src/cache-manager.js

```diff
@@ -28,6 +28,7 @@ class CacheManager {
       'redis', // Redis object where we should cache metadata
       'ensureSSL', // true if we should force only HTTP in redirect links
       'storageProvider', // StorageProvider instance to manage
+      'monitor', // taskcluster-lib-monitor instance
```

Comment:
Since everything in `CacheManager` should be prefixed with `this.id` as I understand it...
I would do:
`this.monitor = this.monitor.prefix(this.id)`

No need to do it later, everything is now consistent :)